src/mtm.js

```diff
@@ -20,7 +20,7 @@ async function authenticate(host, token) {
     console.info(`Successfully generated JWT token.`);
 
     return axios.create({
-      baseURL: `https://eu-vsm.leanix.net/services/vsm/discovery/v1`,
```

Comment:
@aravindmetku as hinted in the comment. I believe this should be parametrized depending on the host. Personally, I'd even expect the following:
1. call MTM with API token 
2. get bearer token
3. decode bearer token and parse `iss` attribute to capture the instance. e.g. eu/us...
4. us region from 3 as input to call vsm-collector

this makes it as user friendly as possible and doing the heavy lifting on the backend.

Reply:
![image](https://user-images.githubusercontent.com/73296015/204340935-4b1ebe9b-ec99-4f48-9b0b-2a398893c744.png)

Reply:
thank you Vince for the insight. The suggestions are incorporated!